app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -70,44 +68,55 @@ class NSIConnectorImpl @Inject() (conf: Configuration, auditor: HTSAuditor, metr
       ()
     )
 
-    val timeContext: Timer.Context = timer.time()
+    val timeContext: Timer.Context = metrics.nsiAccountCreationTimer.time()
```

Comment:
in the unit tests, now we should be able to verify  whether the metrics are working correctly since metrics is injected

Reply:
I was a bit lazy with doing unit tests for metrics since they aren't part of the production logic per se, but I can go ahead and put in unit tests if we really want them there